core/arch/arm/plat-bcm/platform_config.h

```diff
@@ -17,6 +17,14 @@
 
 #define GICD_BASE		0x63c00000
 
+/* Use default AP error logging address */
+#ifndef CFG_BCM_ELOG_AP_UART_LOG_BASE
+#define CFG_BCM_ELOG_AP_UART_LOG_BASE	0x8f110000
+#endif
+
+#define BCM_ELOG_OPTEE_BASE    CFG_BCM_ELOG_AP_UART_LOG_BASE
+#define BCM_ELOG_OPTEE_SIZE    0x10000
```

Comment:
For simplicity: base and size should either be hard-coded here (without `CFG_`), or if they need to be made configurable at compile-time, they should be defined in `conf.mk` as `CFG_BCM_ELOG_OPTEE_BASE ?= 0x8f110000` and `CFG_BCM_ELOG_OPTEE_SIZE ?= 0x10000`. In this case you may use the `CFG_` directly in the code.

Reply:
will move config option `CFG_BCM_ELOG_AP_UART_LOG_BASE` to conf.mk and use it directly in code.